app/main.py

```diff
@@ -1,2 +1,53 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.initial_capacity = 8
```

Comment:
This attribute is not really properly named. The moment you change it, it's no longer initial. Therefore, I'd suggest to rename it to just `capacity`.